vllm/engine/llm_engine.py

```diff
@@ -1240,8 +1248,11 @@ def _process_sequence_group_outputs(
 
         return
 
-    def _process_model_outputs(self, virtual_engine: int,
-                               is_async: bool) -> None:
+    def _process_model_outputs(self,
+                               virtual_engine: int,
+                               is_async: bool,
+                               sampler_output: Optional[SamplerOutput] = None,
+                               is_last_output: bool = False) -> None:
```

Comment:
nit: docstring for `sampler_output` and `is_last_output` can be useful here

Reply:
Good idea, added